server/src/main/java/org/apache/druid/indexing/overlord/IndexerMetadataStorageCoordinator.java

```diff
@@ -157,7 +157,7 @@
    * @param previousSegmentId       previous segment in the series; may be null or empty, meaning this is the first
    *                                segment
    * @param interval                interval for which to allocate a segment
-   * @param shardSpecFactory        shardSpecFactory containing all necessary information to create a shardSpec for the
+   * @param partialShardSpec        shardSpecFactory containing all necessary information to create a shardSpec for the
```

Comment:
Do you want to update the parameter description to match the rename?

Reply:
Thanks, updated.